src/show_dialog.cpp

```diff
@@ -201,7 +201,7 @@ dialog_frame::dimension_measurements dialog_frame::layout(int x, int y, int w, i
 	}
 
 	y -= dim_.title.h;
-	w = std::max<int>(w,std::max<int>(int(dim_.title.w),int(buttons_width)));
+	w = std::max<int>(w,std::max<int>(dim_.title.w,int(buttons_width)));
```

Comment:
It is unnecessary to specify the target type of `std::max()` explicitly, the compiler can figure it out.

`int(buttons_width)` is a C-style cast. Please replace it with `static_cast`.

With those changes, the line becomes

    w = std::max(w, std::max(dim_.title.w, static_cast<int>(buttons_width)));